lib/qemu_store.ml

```diff
@@ -0,0 +1,172 @@
+open Lwt.Infix
+
+let strf = Printf.sprintf
+
+let running_as_root = Unix.getuid () = 0
```

Comment:
Just curious: I can see that this occurs in several places in the code base. But, it seems like it is a misnomer, since afaict the value here gets decided at compile time. So it seem like it should either be

```
let running_as_root () = Unix.getuid () = 0
```

or 

```
let compiled_as_root = Unix.getuid () = 0
```

Any ideas on the intended meaning?